tests/foreman/cli/test_hammer.py

```diff
@@ -196,3 +200,23 @@ def test_positive_disable_hammer_defaults(self):
             Defaults.delete({'param-name': 'organization_id'})
             result = ssh.command('hammer defaults list')
             self.assertTrue(default_org['id'] not in "".join(result.stdout))
+
+    @tier1
+    @run_in_one_thread
+    def test_hammer_host_info_output(self):
+        """Verify re-add of 'owner-id' in `hammer host info` output
+
+        :id: 03468516-0ebb-11eb-8ad8-0c7a158cbff4
+
+        :Steps:
+            1. Update the host with any owner
+            2. Get host info by running `hammer host info`
+
+        :expectedresults: 'owner-id' should be in `hammer host info` output
+
+        :BZ: 1779093
+        """
+        result_list = User.list()
+        Host.update({'owner': settings.server.admin_username, 'owner-type': 'User', 'id': '1'})
+        result_info = Host.info(options={'id': '1', 'fields': 'Additional ' 'info'})
```

Comment:
I believe there was line break and then black changed it:
```suggestion
        result_info = Host.info(options={'id': '1', 'fields': 'Additional info'})
```

Reply:
Updated.